orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/config/Front50ConfigurationProperties.java

```diff
@@ -19,6 +19,30 @@
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+/**
```

Comment:
I can't decide whether I think this comment is helpful enough vs. the chances of it getting stale.  Since these are the higher-precedence properties, I'm pretty tempted to ditch the comment.